### ISSUE TITLE
Should support both SPACE and HTAB as whitespace

### ISSUE THREAD

**Issue by rg2011** [jshttp/media-typer#13]
Currently, the `parse` function can parse media types surrounded by whitespaces, but not tabs:

```
$ node
Welcome to Node.js v20.10.0.
Type ".help" for more information.
> var typer = require('media-typer')
undefined
> typer.parse("text/html ")
MediaType { type: 'text', subtype: 'html', suffix: undefined }
> typer.parse("text/html\t")
Uncaught TypeError: invalid media type
    at Object.parse (/mnt/wsl/tank/projects/type-is/node_modules/media-typer/index.js:117:11)
> 
```

However, I believe `HTAB` should be a valid whitespace character in the context of `Content-Type` header parsing. The specification for the Content-Type header (https://www.rfc-editor.org/rfc/rfc9110#field.content-type) states that:

> The type/subtype MAY be followed by semicolon-delimited parameters (Section 5.6.6) in the form of name/value pairs.

And [section 5.6.6](https://www.rfc-editor.org/rfc/rfc9110#parameter) defines the semicolon-delimited parameters as:

```
  parameters      = *( OWS ";" OWS [ parameter ] )
  parameter       = parameter-name "=" parameter-value
  parameter-name  = token
  parameter-value = ( token / quoted-string )
```

Where `OWS` is defined in [section 5.6.3](https://www.rfc-editor.org/rfc/rfc9110#whitespace) as:

```
  OWS            = *( SP / HTAB )
                 ; optional whitespace
```

So the media-type **might be followed by `HTAB`s  in a valid `Content-Type` header**. If the parser function supports trailing spaces, it should as well support trailing tabs.

**Comment by dougwilson** [comment]
Hello, and thank you for your issue. This module is not made to follow the parsing rules of the HTTP content-typr header, just the RFC that is outlined in the readme.

**Comment by dougwilson** [comment]
I will look to see if RFC 6838 allows tabs or not as well, though I gotta run some errands, but if you find so I can definitely reopen or I will if I find it does.

**Comment by rg2011** [comment]
Hi,

Thanks for reviewing the issue, and sorry for not replying before. I checked the issue on the phone and it seemed I couldn't reply to a closed one. But from desktop I can :D

I understand the motivation to close the issue, and I will not reopen it. I am only commenting to bring your attention to https://github.com/jshttp/type-is/issues/52. I moved the issue there because `type-is` does deal with the `Content-Type` header, so it looks like the proper place to fix it..
